ir/instr.cpp

```diff
@@ -2313,6 +2313,35 @@ void FnCall::print(ostream &os) const {
   os << ')' << attrs;
 }
 
+static void check_tailcall(State &s) {
+  auto *call = static_cast<const Instr *>(s.getCurrentValue());
+  if (!call)
+    return;
+
+  auto it = s.getFn().bbOf(*call).instrs().begin();
+  bool found = false;
+  for (auto e = s.getFn().bbOf(*call).instrs().end(); it != e; ++it) {
```

Comment:
pls don't call bbOf() twice. you can have a single loop to fetch the instruction after the call.

Reply:
Definitely, thanks.